Tests/AsyncHTTPClientTests/HTTPClientInternalTests.swift

```diff
@@ -427,4 +435,21 @@ class HTTPClientInternalTests: XCTestCase {
             XCTFail("wrong message")
         }
     }
+
+    func testResponseConnectionCloseGet() throws {
```

Comment:
how's the test coverage in the connection pool looking here? I think we definitely need more than just one extra test case for something big like a connection pool.

Reply:
@weissi 
Sorry, looks like we need to add back to unit tests that were originally present:
https://github.com/adtrevor/async-http-client/blob/nov14/Tests/AsyncHTTPClientTests/HTTPClientTests.swift#L566
https://github.com/adtrevor/async-http-client/blob/nov14/Tests/AsyncHTTPClientTests/HTTPClientTests.swift#L595

Let me now if you have ideas for additional tests

Reply:
@adtrevor is every line that was added covered by the unit tests? This is how you enable test coverage: https://developer.apple.com/library/archive/documentation/DeveloperTools/Conceptual/testing_with_xcode/chapters/07-code_coverage.html

Reply:
Here are some example of core pieces that aren't hit by any unit test. We should plug those holes. I only looked in `ConnectionPool.swift` only but there are more in other files.

![Screenshot 2019-11-18 at 4 01 17 pm](https://user-images.githubusercontent.com/624238/69068456-b6c5cd80-0a1c-11ea-9d2c-0e422a304b35.png)
![Screenshot 2019-11-18 at 4 01 08 pm](https://user-images.githubusercontent.com/624238/69068457-b6c5cd80-0a1c-11ea-9752-21753219aa67.png)
![Screenshot 2019-11-18 at 4 01 01 pm](https://user-images.githubusercontent.com/624238/69068458-b75e6400-0a1c-11ea-88b4-cf5d908f4d6b.png)
![Screenshot 2019-11-18 at 4 00 24 pm](https://user-images.githubusercontent.com/624238/69068460-b75e6400-0a1c-11ea-92cc-4adab9c13291.png)
![Screenshot 2019-11-18 at 4 00 09 pm](https://user-images.githubusercontent.com/624238/69068461-b75e6400-0a1c-11ea-9a5c-11373456c50d.png)

Reply:
@weissi Thanks, I'm going to check if the tests that were temporarily removed from the PR cover this, I think it definitely should.